src/API/Logs/LogRecordBuilder.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\API\Logs;
+
+class LogRecordBuilder
+{
+    /**
+     * @todo is this needed, or just use LogRecord directly?
```

Comment:
**comment: ** might not be.  Guess we can leave it for now until we decide we don't need it.